pkg/filter/filter_test.go

```diff
@@ -187,6 +187,9 @@ func (s *testFilterSuite) TestCaseSensitive(c *C) {
 	expected := []*Table{{"foo2", "b"}, {"Foo4", "dfoo"}, {"5", "5"}}
 	c.Logf("got %+v, expected %+v", actual, expected)
 	c.Assert(actual, DeepEquals, expected)
+
+	inputTable := &Table{"FOO", "a"}
```

Comment:
How about adding a positive case? for example, `&Table{"foo3", "c"}

Reply:
sure, add one